.github/actions/devcontainer_run_command/action.yml

```diff
@@ -190,5 +193,6 @@ runs:
           -e IS_API_SECURED \
           -e TF_VAR_ci_git_ref="${{ github.ref }}" \
           -e DOCKER_BUILDKIT \
+          -e TF_VAR_debug=${{ inputs.TF_VAR_debug }} \
```

Comment:
Do we still need TF_VAR_debug in this file?

Reply:
fixed